index.html

```diff
@@ -207,6 +382,7 @@ <h1>About me</h3>
                         </ul>
                     </li>
                     <li class="list-style skills">Skills
+                        <span class="span">................</span>
```

Comment:
Kindly increase the font-size for the Àbout-me`heading, use the right font-type for the whole of your project and align the button to the left as it is in the design you were given 👍 

Figma design :
![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/02a7cb2d-cf88-4807-852d-5c2a9b3597f5)

Your design :
![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/a3da1a09-7d23-49d0-980d-4b9088856bac)